src/readinglistbar.cpp

```diff
@@ -60,13 +70,44 @@ void ReadingListBar::setupList()
     }
 }
 
-void ReadingListBar::on_itemActivated(QListWidgetItem* item)
+void ReadingListBar::onItemClicked(QListWidgetItem* item) // Receives single clicks
+{
+    clickKind = 1;
+    QTimer::singleShot(QApplication::doubleClickInterval(), [=]() { // Give time for a double click be handled
+        if (clickKind == 1) {
+            openUrl(item, false);
+        }
+    });
+}
+
+void ReadingListBar::onItemDoubleClicked(QListWidgetItem* item) // Receives double clicks
+{ 
+    lastDoubleClickTime = QDateTime::currentDateTime();
+    openUrl(item, true);
+}
+
+void ReadingListBar::onItemPressed(QListWidgetItem* item, Qt::MouseButtons buttons) // Receives single and middle click
+{
+    if (buttons & Qt::MiddleButton) {
+        openUrl(item, true);
+    }
+}
+
+void ReadingListBar::onItemActivated(QListWidgetItem* item, Qt::MouseButtons buttons) // Receives double click and activation key
```

Comment:
Fix the comment (activation via mouse is platform dependent and can be either a single click or a double click)